deploy/10-dependencies-sfrax-oracle.ts

```diff
@@ -5,16 +5,15 @@ import { HardhatRuntimeEnvironment } from "hardhat/types";
 import { ADDRESSES } from "../helpers/deploymentConfig";
 
 const func: DeployFunction = async ({ getNamedAccounts, deployments, network }: HardhatRuntimeEnvironment) => {
+  const networkName: string = network.name === "hardhat" ? "sepolia" : network.name;
```

Comment:
I wouldn't alias the hardhat network name, instead I'd look at handling the case where the network name is hardhat. This is why I recommend fetching deployments instead of hardcoding them or importing them.

A compromise would be to make `ADDRESSES` a function that would use the hardcoded or imported addresses on live networks and fetch them on hardhat.

Reply:
that also works 👍